types/v56/openapi.go

```diff
@@ -650,3 +650,25 @@ type VcenterDistributedSwitch struct {
 	BackingRef    OpenApiReference `json:"backingRef"`
 	VirtualCenter OpenApiReference `json:"virtualCenter"`
 }
+
+// OpenApiMetadataEntry represents a metadata entry in VCD.
+type OpenApiMetadataEntry struct {
+	ID           string                  `json:"id,omitempty"`         // UUID for OpenApiMetadataEntry. This is immutable
+	IsPersistent bool                    `json:"persistent,omitempty"` // Persistent entries can be copied over on some entity operation, for example: | Creating a copy of an Org VDC | Capturing a vApp to a template | Instantiating a catalog item as a VM
```

Comment:
What's with the pipe "|" separators? :)

Reply:
Not my invention! Coming from https://developer.vmware.com/apis/vmware-cloud-director/latest/data-structures/MetadataEntry/ 😄 
I can change to commas.

Reply:
I see :) Small nit: I'd add a dot at the end of "etc."

Reply:
Done, thanks!